app/__init__.py

```diff
@@ -36,13 +33,15 @@
 except OSError:
     pass
 
+
+@app.context_processor
```

Comment:
This makes a variable, `google_analytics_id` available to the front-end templates